server/controllers/user.controller.js

```diff
@@ -293,6 +293,21 @@ class UserController {
       return Response(res, 201, message);
     }
   }
+
+  /**
+   * @description get list of user functionality
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} list of user profiles
+   */
+  static async getUsers(req, res) {
+    const users = await User.findAndCountAll({
+      attributes: {
+        exclude: ['password', 'istokenreset', 'subscribed']
+      }
+    });
+    return Response(res, 200, 'Successfully retrieved users', users);
+  }
```

Comment:
Good job. Since we are using findandcount, why not include the number of users found as part of the message?
For example, 'Successfully retrieved 13 users.

Reply:
Yeah it will be good to implement this but `Successfully retrieved users` is being used in the test.